src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/SubmissionRenderingErrorNotification.java

```diff
@@ -0,0 +1,32 @@
+package fi.aalto.cs.apluscourses.intellij.notifications;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import org.jetbrains.annotations.NotNull;
+
+public class SubmissionRenderingErrorNotification extends Notification {
+
+  @NotNull
+  private final Exception exception;
+
+  /**
+   * Construct a notification informing the user that an error occurred while attempting to render
+   * a submission.
+   */
+  public SubmissionRenderingErrorNotification(@NotNull Exception exception) {
+    super(
+        "A+",
+        "Failed to open submission",
+        "Failed to open submission. Please open the submission in the A+ web interface. Error "
```

Comment:
I propose to modify the text a bit, perhaps "Failed to open the submission page from the IDE. Please open the submission in the A+ web interface (browser).". unsure if we actually need an error message shown to the user?

additionally, what if that same error message (notification) contains a hyperlink to the corresponding A+ page (a clickable link)?